core/src/test/java/com/linecorp/armeria/server/ServerBuilderTest.java

```diff
@@ -684,4 +684,157 @@ void exceptionReportIntervalMilliSeconds() {
                       .service("/", (ctx, req) -> HttpResponse.of(HttpStatus.OK))
                       .build());
     }
+
+    @Test
+    void virtualHostContextPath() {
+        final Server server = Server
+                .builder()
+                // 1
+                .port(8888, SessionProtocol.HTTP)
+                .service("/foo", (ctx, req) -> HttpResponse.of(HttpStatus.OK))
+                .port(4848, SessionProtocol.HTTP)
+                .service("/hello", (ctx, req) -> HttpResponse.of(HttpStatus.OK))
+                // 2
+                .virtualHost("*.foo.com:8888")
+                .contextPath("/api/v1")
+                .service("/bar", (ctx, req) -> HttpResponse.of(HttpStatus.OK))
+                .service("/good", (ctx, req) -> HttpResponse.of(HttpStatus.OK))
+                .and()
+                .virtualHost("*.bar.com:4848")
+                .contextPath("/api/v2")
```

Comment:
We'll be introducing a `contextPath` API which serves APIs for a subset of services.

What do you think of renaming this to `ServerBuilder#baseContextPath()`, `VirtualHostBuilder#baseContextPath()` instead?

Reply:
No problem.